todo_app/templates/index.html

```diff
@@ -11,8 +11,19 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in list_items %}
+            <li> Number {{ item.id }} - {{item.title}} - {{item.status}}  </li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+ 
+  
+  <form action="/additem" method="POST">
+  <label for="title">Add Item:</label><br>
+  <input type="text" id="id" name="title" value=""><br>
```

Comment:
Minor, but adding a `required` attribute here would add a little bit of nice browser-side validation so that users don't accidentally submit an empty input

Reply:
Thanks Jack